react/utils/pixel.ts

```diff
@@ -10,15 +13,16 @@ export enum EventType {
 }
 
 export function handleProductClick(push: (data: any) => void, page: string) {
-  return (productId: string, position: number) =>
+  return (products: ISearchProduct[], id: string, position: number) =>
```

Comment:
this didn't make much sense to me..
you are adding a `products` prop, but it will always send only one product to the pixel manager, which is the same as the product id. couldn't you just pass here the product you are going to use?